app/views/results/common/_flexible_marks_summary_table.html.erb

```diff
@@ -26,7 +26,7 @@
     <tr class="<%= cycle('mark_summary_holderA', 'mark_summary_holderB') %>" id="<%= "test_script_summary" %>">
         <td><%=t("summary_table.test_script_title")%></td>
         <td><%= h(@assignment.total_test_script_marks * 1.0) %></td>
-        <td><span id="test_script_summary_mark"><%= h(@submission.result.get_total_test_script_marks * 1.0) %></span> / <%=  h(@assignment.total_test_script_marks * 1.0) %></td>
+        <td><span id="test_script_summary_mark"><%= h(@result.get_total_test_script_marks * 1.0) %></span> / <%=  h(@assignment.total_test_script_marks * 1.0) %></td>
```

Comment:
Yes, I was getting "undefined method `result'" from _rubric_marks_summary_table.html.erb, so I changed this file too at the same time. If you look at view/results/marker/_marker_panes.erb.html at line 133, only result get passed in as local variables.